packages/design-system/.storybook/preview.js

```diff
@@ -297,6 +297,7 @@ export const parameters = {
 					'Layout',
 					'Link',
 					'Loading',
+					'Messages',
```

Comment:
```suggestion
					'Message',
```

Reply:
Is it the reason it's not well sorted ? 🤔